samples/Barrel.c

```diff
@@ -16,6 +16,7 @@
 #include <math.h>
 #include "Tilengine.h"
 #include "Simon.h"
+#include "../src/sdl/SDL2/SDL.h"
```

Comment:
It is not possible to compile this PR as it has `#include` directives to SDL header files that are not in the project.

You should tell your compiler those directives and not use absolute paths to those files from here.

Reply:
I just noticed there is a GetTicks (TLN_GetTicks) function built into Tilengine, I'll try with this one.

Reply:
Okay so I removed thoses includes and used TLN_GetTicks instead of SDL_GetTicks. It should work now.